cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -202,6 +202,25 @@ def _get_mapping_dict_feature_reference(
 
         return mapping_dict
 
+    def _get_mapping_dict_feature_biotype(self, ids: List[str]) -> Dict[str, str]:
+        """
+        Creates a mapping dictionary of feature IDs and biotype ("gene" or "spike-in")
+
+        :param list[str] ids: feature IDs use for mapping
+
+        :return a mapping dictionary: {id: "gene", id: "spike-in", ...}
+        :rtype dict
+        """
+        mapping_dict = {}
+
+        for i in ids:
+            if i.startswith('ERCC'):
+                mapping_dict[i] = "spike-in"
+            else:
+                mapping_dict[i] = "gene"
```

Comment:
In the future we are likely to accept "protein" as `feature_type` so let's not make it just an else for gene, raising an error ensures that future developers won't let this slip

```suggestion
            elif i.startswith('ENS'):
                 mapping_dict[i] = "gene"
            else:
               raise ValueError (f"{i} is not a recognized `feature_name` and cannot be assigned a `feature_type`)
```